packages/lib/src/helpers/copyContract.ts

```diff
@@ -0,0 +1,38 @@
+import { promisify } from 'util';
```

Comment:
Are you using prepare_ts.sh or otherwise git mv instead of mv? That way we get more atomic diffs like this one: https://github.com/zeppelinos/zos/pull/443/commits/56bc171da4ac19136169d3b105b3e08e9a092219

Reply:
Otherwise LGTM!

Reply:
> Are you using prepare_ts.sh or otherwise git mv instead of mv? That way we get more atomic diffs like this one: [56bc171](https://github.com/zeppelinos/zos/commit/56bc171da4ac19136169d3b105b3e08e9a092219)

I will use it from now on :slightly_smiling_face: